### ISSUE TITLE
Validation loss goes to infinity

### ISSUE THREAD

**Issue by soummyaah** [tkipf/relational-gcn#11]
I have a large graph with 7000 nodes and 20 million triples with 110 class types and 59 relation types. Somewhere along the training, the validation loss suddenly becomes infinite. We also tried this on a much smaller graph with 7000 nodes and 117k triples with 110 class types and 10 relation types. The validation loss somehow kept increasing instead of decreasing until it went to infinity suddenly.
Any insights into what might be causing it and how to debug? Is it just an issue of the graph size or could something else be wrong?

**Comment by tkipf** [comment]
I have not experienced this myself with this code release, have you maybe made some changes to the implementation? 

Maybe it is also worth trying this PyTorch implementation: https://docs.dgl.ai/tutorials/models/1_gnn/4_rgcn.html

**Comment by 201528014227051** [comment]
the infinity maybe related to the dataself
